pkg/reboot/calculator.go

```diff
@@ -145,10 +148,11 @@ func (s *safeTimeCalculator) calcNumOfBatches() int {
 			numberOfBatches++
 		}
 	}
-	//In order to stay on the safe side taking the largest calculated batch number (capped at 11)
-	if s.highestCalculatedBatchNumber < numberOfBatches {
-		s.highestCalculatedBatchNumber = numberOfBatches
-	}
-	return s.highestCalculatedBatchNumber
+	return numberOfBatches, nil
+}
 
+func (r *calculator) getConfig(k8sClient client.Client, ctx context.Context, namespace string) (*v1alpha1.SelfNodeRemediationConfig, error) {
```

Comment:
please remove k8sClient from method params

Reply:
the complete method can be removed 🙈